packages/buckets/src/buckets.ts

```diff
@@ -21,6 +21,20 @@ export interface PushPathResult {
 
 /**
  * Buckets is a web-gRPC wrapper client for communicating with the web-gRPC enabled Textile Buckets API.
+ * @example
```

Comment:
if we add some of these inline examples to the client lib, they should start getting picked up too

Reply:
nice